index.rst

```diff
@@ -130,6 +130,7 @@ Sensor Components
     Resistance, components/sensor/resistance, omega.svg
     Rotary Encoder, components/sensor/rotary_encoder, rotary_encoder.jpg
     SDS011 Sensor, components/sensor/sds011, sds011.jpg
+    SCD30, components/sensor/scd30, scd30.jpg
```

Comment:
Try compressing this file further - 65.9kb is quite a lot for the main page, the "contributing" page has a command that will compress it down even more. If you can't install that program, I can also compress it for you.

Reply:
Thanks, ive cropped and optimized the photo to 15.5kB and added the 4-pin designation